### PR TITLE
use ColorPropType everywhere there is a color prop on TextInput

### PR DESCRIPTION
**Pull request by erickreutz**
If using colors that are not strings react native would warn you of the problem. I made some changes so color props use `ColorPropType` instead of `PropTypes.string`.

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @Bhullnatik and @dmmiller to be potential reviewers.

File: Libraries/Components/TextInput/TextInput.js [hunk 1]
```diff
@@ -26,6 +26,7 @@ var TimerMixin = require('react-timer-mixin');
 var TouchableWithoutFeedback = require('TouchableWithoutFeedback');
 var UIManager = require('UIManager');
 var View = require('View');
+var ColorPropType = require('ColorPropType');
```
Comment (ide) [inline Libraries/Components/TextInput/TextInput.js]: Maintain alphabetical ordering

Comment (erickreutz) [inline Libraries/Components/TextInput/TextInput.js]: Done. Thanks!

**Comment by ide** [comment]
LGTM, let's wait for the CircleCI tests

**Comment by erickreutz** [comment]
@ide how long does it usually take?

**Comment by javache** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-6** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/596497887178829/int_phab) to review.

**Pull request closed without merging**